.github/workflows/pr-vrt-comment.yml

```diff
@@ -34,134 +31,66 @@ jobs:
       # downloaded artifacts will contain screenshots from affected project including 'screenshots-report.json' which contains proper image mappings for affected project
       # - see @{link file://./../scripts/prepare-vr-screenshots-for-upload.js#43}
       # - see @{link file://./pr-vrt.yml#56}
+
       - uses: actions/download-artifact@v4
         with:
           name: vrscreenshot
           path: ./screenshots
           run-id: ${{ github.event.workflow_run.id }}
           github-token: ${{ secrets.GITHUB_TOKEN }}
 
-      - uses: actions/download-artifact@v4
-        with:
-          name: pr-number
-          path: ./results
-          run-id: ${{ github.event.workflow_run.id }}
-          github-token: ${{ secrets.GITHUB_TOKEN }}
-
-      - name: Load PR number
-        uses: actions/github-script@v7
-        id: pr_number
-        with:
-          script: |
-            const run = require('./.github/scripts/validate-pr-number');
-            const result = run({filePath:'results/pr.txt'});
-            return result;
-          result-encoding: string
-
-      - name: VR App - Create Policy
-        run: |
-          echo "MAKE THIS STEP WORK"
-
       - name: Login via Azure CLI
         uses: azure/login@a65d910e8af852a8061c627c456678983e180302 # v2.2.0
         with:
-          client-id: ${{ secrets.AZURE_CLIENT_ID }}
+          client-id: ${{ secrets.AZURE_VRT_CLIENT_ID }}
           tenant-id: ${{ secrets.AZURE_TENANT_ID }}
           subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
 
-      - name: Fetch Access Token
-        run: |
-          echo "ACCESSTOKEN=$(az account get-access-token --query accessToken --output tsv)" >> $GITHUB_ENV
-
-      - name: Run screenshotdiff
+      - name: Run VR Approval CLI
+        uses: actions/github-script@v7
         env:
-          VR_APP_API_URL: 'https://vrapprovaldev2.azurewebsites.net/api/'
-          STORAGE_ACCOUNT_ID: 'https://onejstestartifactsprod.blob.core.windows.net/'
-          SYSTEM_ACCESSTOKEN: ${{ env.ACCESSTOKEN }}
-          TENANT_ID: ${{ secrets.AZURE_TENANT_ID }}
-          PRINCIPAL_CLIENT_ID: ${{ secrets.AZURE_CLIENT_ID }}
-          SERVICE_CONNECTION_ID: ${{ secrets.ADO_VRT_SERVICE_CONNECTION_ID }}
           GITHUB_API_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: |
-          echo "MAKE THIS WORK"
-          npx vr-approval-cli@0.4.11 run-diff --screenshotsDirectory ./screenshots --buildType pr --clientType "FLUENTUI" --threshold '0.04' --cumThreshold '1'
+          PRINCIPAL_CLIENT_ID: ${{ secrets.AZURE_VRT_CLIENT_ID }}
+        with:
+          script: |
+            const data = {
+              GITHUB_RUN_ID: context.payload.workflow_run.id,
+              GITHUB_REF: `refs/pull/${context.payload.workflow_run.pull_requests[0].number}/merge`,
```

Comment:
are you sure this payload will have any data ? AFAIR we tried this some time ago to avoid the PR number upload / download but it was empty.

Reply:
<img width="344" alt="image" src="https://github.com/user-attachments/assets/75a1e110-0b48-4c14-9f10-145ab4c280d8" />

check Debug step output https://github.com/microsoft/fluentui/actions/runs/13947864176/job/39039278471

Reply:
It might be due to permissions? Sending you a link to a private github repo on teams which has this working.